1.py

```diff
@@ -0,0 +1,2 @@
+for i in range (1,43):
```

Comment:
For this task, according to the context of the lesson, you should use arithmetic operations with strings. Could you update this part using the multiplication of string and integer?